<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ROSVisualizer</class>
 <widget class="QWidget" name="ROSVisualizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="7,0">
     <item>
      <widget class="ZoomableView" name="rosVisual"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>160</height>
          </size>
         </property>
         <property name="title">
          <string>机器人坐标</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>25</y>
            <width>151</width>
            <height>101</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="labelX">
               <property name="text">
                <string>x:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="robPosX">
               <property name="text">
                <string>0.00m</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="labelY">
               <property name="text">
                <string>y:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="robPosY">
               <property name="text">
                <string>0.00m</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="labelZ">
               <property name="text">
                <string>z:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="robPosZ">
               <property name="text">
                <string>0.00m</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="labelYaw">
               <property name="text">
                <string>yaw:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="robPosYaw">
               <property name="text">
                <string>0.00°</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item alignment="Qt::AlignBottom">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QWidget" name="layoutWidget1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>25</y>
            <width>151</width>
            <height>121</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="locateBtn">
             <property name="text">
              <string>定位机器</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveMapBtn">
             <property name="text">
              <string>保存地图</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearBtn">
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomableView</class>
   <extends>QGraphicsView</extends>
   <header>rosvisualizer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
